docs/Consuming.md

```diff
@@ -50,9 +50,26 @@ The `CONSUMER` block defines:
     * 0,1,2
     * 0-2
     * 0,2-3
+ * `key-format` : [deserializer](#Deserializer) to use for the key.
+ * `value-format` : [deserializer](#Deserializer) to use for the value. 

```

Comment:
add `*[optional]*` at the end

Reply:
explain that, by default, an UTF-8 string deserializer is used

Reply:
fixed. I have add string as MessageFormat too.